README.md

```diff
@@ -114,7 +135,8 @@ Request
   "info": {
     "alias": "Nice Orange",
     "deviceModel": "Samsung", // nullable
-    "deviceType": "mobile" // mobile | desktop | web
+    "deviceType": "mobile", // mobile | desktop | web
```

Comment:
device type can allows more device form? e.g:

- `headless` (command-line)

   (i'm is developing)

   e.g:

   ```
   localsend-go send <send the file path> <fingerprint or host-ip>
   localsend-go receive <saved to file path> <fingerprint or host-ip>
   ```

- `server` (receive only)

   the form (work mode), can help users backup file to NAS or storage plaform

   simplify upload to desktop computer, then manually upload file to NAS or storage platform

Reply:
Sure, we can add `headless` and `server` there.